src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import, print_function
+
+import logging
+
+from datetime import timedelta
+
+from django.core.cache import cache
+from django.utils import timezone
+
+from sentry.eventstore.models import Event
+from sentry.eventstore.processing import event_processing_store
+from sentry.models import Project, Release
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils.committers import get_serialized_event_file_committers
+
+PREFERRED_GROUP_OWNERS = 3
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+
+logger = logging.getLogger("tasks.groupowner")
+
+
+@instrumented_task(
```

Comment:
not sure if this runs on the right queue if the queue is omitted. I think it may now run on worker-glob, but I'd check with ops if that's fine

putting it into a separate queue might ehlp with killing those tasks if they go rogue

Reply:
+1 to giving this a unique queue, it's expensive to calculate suspect commits.